src/api/firebase.js

```diff
@@ -239,10 +240,12 @@ export async function updateItem(listPath, itemID, isChecked) {
 	}
 }
 
-export async function deleteItem() {
-	/**
-	 * TODO: Fill this out so that it uses the correct Firestore function
-	 * to delete an existing item. You'll need to figure out what arguments
-	 * this function must accept!
-	 */
+export async function deleteItem(listPath, itemID) {
+	// Note - firebase deleteDoc will remove the entire collection if the item being removed is the last in said collection!
```

Comment:
question: Just curious if that has any implications for any other functionality? I tried adding one item to one of my own lists, deleting it, and re-adding it and it didn't seem to break anything 🤞🏻

Reply:
From some further digging, we noticed the pattern from the dinner list, and others that get created. Our lists are stored as documents within each of our user id collections. These documents then contain the 'items' items, which then has the documents for each item in the list. Which theoretically means it only removes the 'items' collection, but doesn't touch the list doc itself. AFAIK, there shouldn't be any functionality implications for now.

Reply:
> there shouldn't be any functionality implications for now

I think it's mostly an aspect that will be useful for us to know as the developers of this project, as we look at documentation / seek to build on this structure that we have!